src/index.html

```diff
@@ -5,8 +5,465 @@
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Catalog</title>
     <link rel="stylesheet" href="./styles/main.scss">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Inter:wght@400;700&family=Montserrat:wght@400;700&family=Roboto:wght@500&display=swap" rel="stylesheet">
   </head>
   <body>
-    <h1>Catalog</h1>
+    <header class="header">
+      <a href="#" class="header__logo">
+        <img
+          src="/src/images/logo.png"
+          alt="moyo-logo"
+          class="header__logo-link"
+        >
+      </a>
+      <nav class="header__nav">
+        <ul class="header__nav-list">
+          <li class="header__item">
+            <a
+              href="https://www.moyo.ua/"
+              class="header__nav-link is-active"
+              data-qa="nav-hover"
+            >
+              Apple
+            </a>
+          </li>
+          <li class="header__item">
```

Comment:
Add a blank line between the links, and fix the classes for the tag li
```suggestion
          </li>
          
          <li class="nav__item">
```

Reply:
Тоді вкладеність не працює і помилка виникає, в чаті навпаки порадили використання header__item, оскульки це в хедері